.circleci/config.yml

```diff
@@ -1,3 +1,51 @@
+e2eTest: &e2eTest
+    machine: true
+    working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/chart-operator
+    steps:
+    - checkout
+
+    - attach_workspace:
+        at: .
+
+    - run: |
+        wget -q $(curl -sS https://api.github.com/repos/giantswarm/e2e-harness/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+        chmod +x ./e2e-harness
+
+    - run:
+        name: setup localkube in circleCI machine
+        environment:
+          MINIKUBE_VERSION: v0.25.0
+        command: ./e2e-harness localkube
+
+    - run: ./e2e-harness setup --remote=false
+
+    - run:
+        name: start CNR server
+        command: |
+          docker run \
+                 --name cnr-server \
+                 --network host \
+                 -d -t quay.io/giantswarm/docker-cnr-server
```

Comment:
Does it make sense to add a tag/version to the docker image? When merging in the docker-cnr-server repo, you wouldn't know that stuff breaks here right?

Reply:
Very good point, I've added a tag to the image and asked for it here :+1: